src/System.Security.Cryptography.X509Certificates/tests/X509FilesystemTests.Unix.cs

```diff
@@ -15,12 +15,14 @@ namespace System.Security.Cryptography.X509Certificates.Tests
     [Collection("X509Filesystem")]
     public static class X509FilesystemTests
     {
-        // #9293: Our Fedora and Ubuntu1610 CI machines use NTFS for "tmphome", which causes our filesystem permissions checks to fail.
-        private static bool IsReliableInCI { get; } = ChainTests.IsReliableInCI;
+        private static bool CanModifyStores { get; } = TestEnvironmentConfiguration.CanModifyStores;
+        private static bool RunManualTests { get; } = TestEnvironmentConfiguration.RunManualTests;
 
-        [ActiveIssue(12833, TestPlatforms.AnyUnix)]
-        [Fact]
         [OuterLoop]
+        // This test is a bit too flaky to be on in the normal run, even for OuterLoop.
+        // It can fail due to networking problems, and due to the filesystem interactions it doesn't
+        // have strong isolation from other tests (even in different processes).
+        [ConditionalFact(nameof(RunManualTests))]
```

Comment:
I've defined the environment variable on my machines; so I run it several times a day.  I'm also fine with just deleting the test; but there're a bit too many variables to keep it sufficiently reliable in CI (the fact that it needs to delete a file then have neither a parallel test nor parallel job write that same file again while then needing the network to be reliable during that window is (apparently) a bit much to ask for)